js/localStorage.js

```diff
@@ -0,0 +1,33 @@
+const inputName = document.querySelector('#name');
+const inputEmail = document.querySelector('#mail');
+const inputText = document.querySelector('#msg');
```

Comment:
[OPTIONAL] I think it could be cool if the data (all of it or just the message 🤔) gets deleted from the local storage when the user finishes sending the form, that way the form would be clean once he has finished sending a message. This could result in a better user experience. What do you think?